### PR TITLE
[Doc enhancement][Added a note in Messaging extensions][2038438]

### PR DESCRIPTION
**Pull request by v-habibaumme**
Added a note in define-action-command and  define-search-command files

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/action-commands/define-action-command.md](https://github.com/MicrosoftDocs/msteams-docs/blob/58c93c4ecb78ff782597f94a072c3cc48bf55908/msteams-platform/messaging-extensions/how-to/action-commands/define-action-command.md) | 89 | 26 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/33a86fed-b1c9-49a0-9301-258780e887a4) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/how-to/search-commands/define-search-command.md](https://github.com/MicrosoftDocs/msteams-docs/blob/58c93c4ecb78ff782597f94a072c3cc48bf55908/msteams-platform/messaging-extensions/how-to/search-commands/define-search-command.md) | 89 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9ba9e5fe-d52b-45a1-81f1-bb22d24cfc45) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/action-commands/define-action-command.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c60d515a40a294f700cd83eb5838221356b4ac0e/msteams-platform/messaging-extensions/how-to/action-commands/define-action-command.md) | 90 | 24 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/656bc907-301f-418b-8c51-c0c47998d0f3) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/how-to/search-commands/define-search-command.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c60d515a40a294f700cd83eb5838221356b4ac0e/msteams-platform/messaging-extensions/how-to/search-commands/define-search-command.md) | 89 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d003e8de-cc36-408c-aa30-98c2216d37b9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/search-commands/define-search-command.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1b0a1e5bba935275c2bdf721268a8907bfd01c39/msteams-platform/messaging-extensions/how-to/search-commands/define-search-command.md) | 89 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ca9bd4e2-0ada-4684-9123-abc0fe704271) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/what-are-messaging-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1b0a1e5bba935275c2bdf721268a8907bfd01c39/msteams-platform/messaging-extensions/what-are-messaging-extensions.md) | 89 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/28859497-6d1d-46ad-8655-9f68b604c5ee) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/messaging-extensions/how-to/search-commands/define-search-command.md [hunk 1]
```diff
@@ -22,6 +22,9 @@ The search command is invoked from any one or both of the following locations:
 * Compose message area: The buttons at the bottom of the compose message area.
 * Command box: By @mentioning in the command box.
 
+   > [!NOTE]
+   > @mention is no longer supported for messaging extension.
```
Comment (stkong-sdk) [inline msteams-platform/messaging-extensions/how-to/search-commands/define-search-command.md:26]: Can we change the note to "@mentioning message extensions is no longer supported in the compose box."

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/how-to/search-commands/define-search-command.md](https://github.com/MicrosoftDocs/msteams-docs/blob/49ad3ba4717fe792acbfe02dce5ed23396140dbf/msteams-platform/messaging-extensions/how-to/search-commands/define-search-command.md) | 89 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fdf99818-8fd9-4b7d-9129-f26d8a9c6699) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/what-are-messaging-extensions.md](https://github.com/MicrosoftDocs/msteams-docs/blob/49ad3ba4717fe792acbfe02dce5ed23396140dbf/msteams-platform/messaging-extensions/what-are-messaging-extensions.md) | 89 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3aba6722-6907-46ff-b98b-fc299f57caa3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**